dev/PushNotifications/PushNotificationsLongRunningTask/platform.cpp

```diff
@@ -240,6 +246,16 @@ const std::wstring NotificationsLongRunningPlatformImpl::BuildAppIdentifier(std:
     return guidStr.get();
 }
 
+void NotificationsLongRunningPlatformImpl::AddComServerClsid(std::wstring const& appId, winrt::guid const& comServerClsid)
+{
+    m_comServerClsidStorage.Values().Insert(appId, winrt::box_value(comServerClsid));
```

Comment:
No locks

Reply:
Actually is a lock needed at all? It seems to be called only from a locked context.